test/unit/app/common/state/ledgerStateTest.js

```diff
@@ -622,4 +623,32 @@ describe('ledgerState unit test', function () {
       assert.equal(result, 'corrupted')
     })
   })
+
+  describe('getVerifiedPublisherLocationProp', function () {
```

Comment:
minor: it would be good to have a test case for a URL without a protocol, ex: `brianbondy.com`

Reply:
This got me thinking: From everything I've seen in this so far. URLs already have the protocol resolved before they get to this point. @diracdeltas or @NejcZdovc, do you know of any exceptions where URLs don't have a protocol already before getting to the publisherToggle renderer? If not, then I might have been overthinking ```shortenUrlToDomainTLD``` logic

Reply:
@jasonrsadler I don't know of any examples but I also am not familiar with this code path at all

Reply:
not that I am aware